### DISCUSSION TITLE
Welcome to discussions!

### DISCUSSION THREAD

**Discussion by tbabej** [GothenburgBitFactory/taskwarrior#2378] [General]
We are starting Github discussions as an alternative to setting up (currently a work-in-progress) user mailing list.

**Comment by djmitche** [comment]
What about https://groups.google.com/g/taskwarrior-user?

**Comment by tbabej** [comment]
@djmitche Good point! My personal goal would be to migrate away from the Google hosted mailing list to a self-hosted Mailman3+Hyperkitty solution.

Happy to discuss that (perhaps in a separate thread?) if that comes across as unnecessary, as one could argue that migration it's going to churn users. That said, the user group is not particularly active, was not getting a lot of attention (as it was introduced right at the beginning of the taskwarrior's recent "quiet" period).

**Reply by djmitche** [reply]
It certainly has been a quiet list :)

IMHO mailing lists are kind of passé, so just using GH discussions sounds good to me.  And a "migration" from Googly groups to GH is probably no more lossy than a migration from groups to mailman.
